src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+import uuid
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        max_length=40,
+        help_text=_(
+            "Unieke (technische) identificatiecode van de betrokkene bij klantcontact."
+        ),
+    )
+    nummer = models.PositiveIntegerField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[MaxValueValidator(9999999999)],
+        blank=False,
+    )
+    interne_notitie = models.CharField(
+        _("Interne notitie"),
+        help_text=_(
+            "Mededelingen, aantekeningen of bijzonderheden over de partij, bedoeld voor intern gebruik."
+        ),
+        max_length=1000,
+        blank=True,
+    )
+    soort_partij = models.CharField(
+        _("Soort partij"),
+        help_text=_("Geeft aan van welke specifieke soort partij sprake is."),
+        max_length=14,
+        choices=SoortPartij.choices,
+        blank=False,
+    )
+    indicatie_geheimhouding = models.BooleanField(
+        _("Indicatie geheimhouding"),
+        help_text=_(
+            "Geeft aan of de verstrekker van partijgegevens heeft aangegeven dat "
+            "deze gegevens als geheim beschouwd moeten worden."
+        ),
+        blank=False,
+    )
+    voorkeurskanaal = models.CharField(
+        _("Voorkeurskanaal"),
+        help_text=_(
+            "Kanaal dat de partij bij voorkeur gebruikt voor contact met de gemeente."
+        ),
+        max_length=50,
```

Comment:
From the specs, I don't really understand what voorkeurskanaal should be :thinking: I see that it should have format 'Kanaal' but I can't find this? How did you choose the max length?

Reply:
It is not specified as a `primitief datatype` but I did find it referenced here, so we can atleast maintain that (the length is the same for `Kanaal`.

![Screenshot from 2023-10-12 17-53-52](https://github.com/maykinmedia/open-klant/assets/101265650/19e234e5-00c4-4eca-b090-b976e4b5e9cf)